.github/workflows/cve_checks.yml

```diff
@@ -32,34 +39,16 @@ jobs:
           export VERSION=$(./mvnw -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
           echo "version=${VERSION}" >> $GITHUB_OUTPUT
 
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v3
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v3
-
-      - name: Cache Docker layers
-        uses: actions/cache@v4
-        with:
-          path: /tmp/.buildx-cache
-          key: ${{ runner.os }}-buildx-${{ github.sha }}
-          restore-keys: |
-            ${{ runner.os }}-buildx-
-
-      - name: Build docker image
-        uses: docker/build-push-action@v5
+      - name: Build Docker Image
+        uses: ./.github/workflows/docker_build.yml
         with:
-          builder: ${{ steps.buildx.outputs.name }}
-          context: api
-          platforms: linux/amd64
-          push: false
-          load: true
-          tags: |
-            ghcr.io/kafbat/kafka-ui:${{ steps.build.outputs.version }}
-          build-args: |
-            JAR_FILE=api-${{ steps.build.outputs.version }}.jar
-          cache-from: type=local,src=/tmp/.buildx-cache
-          cache-to: type=local,dest=/tmp/.buildx-cache
+          tag: ${{ steps.extract_branch.outputs.tag }}
+          version: ${{ steps.build.outputs.version }}
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: eu-central-1
+          repository: ghcr.io/kafbat/kafka-ui
```

Comment:
given there are aws-related properties above, this looks like it should be AWS-related, why ghcr.io?